manifests/course/puppetize.pp

```diff
@@ -14,6 +14,10 @@
       mode  => '0644',
     }
 
+    class { 'classroom':
```

Comment:
I tried adding some code to classroom::virtual to manage time servers, but it caused some errors. Then I realized I'd need gitea as well. I'll go back and look at it again and see if I can figure out a different solution.